client/packages/invoices/src/Prescriptions/api/operations.graphql

```diff
@@ -50,6 +50,15 @@ fragment PrescriptionRow on InvoiceNode {
     firstName
     lastName
   }
+
+  currency {
+    id
+    code
+    rate
+    isHomeCurrency
+  }
+
+  currencyRate
```

Comment:
Is this an unrelated change?

Reply:
Ah yes sorry. Mark also fixed this in one of his PRs. The currency changes broke generation